it/controllers/AttachmentsControllerISpec.scala

```diff
@@ -34,35 +33,6 @@ class AttachmentsControllerISpec extends IntegrationStubbing {
   val attachmentsUrl: String = routes.AttachmentsController.getAttachmentList(testRegId).url
 
   s"GET /:regId/attachments " must {
-    "return OK with identityEvidence for a NETP vat scheme" in new Setup {
```

Comment:
I would add at least one test here for the get attachments endpoint returning something